### PR TITLE
gosbom: init at 0.80.0

### PR DESCRIPTION
**Pull request by gitworkflows**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [x] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Janik-Haag — CHANGES_REQUESTED** [review]
You are at least missing a package entry in `pkgs/top-level/all-packages.nix`. Can you add this?

File: pkgs/tools/admin/gosbom/default.nix [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    homepage = "https://github.com/nextlinux/gosbom";
+    changelog = "https://github.com/nextlinux/gosbom/releases/tag/v${version}";
+    description = "CLI tool and library for generating a Software Bill of Materials from container images and filesystems";
+    longDescription = ''
+      A CLI tool and Go library for generating a Software Bill of Materials
+      (SBOM) from container images and filesystems. Exceptional for
+      vulnerability detection when used with a scanner tool like GoVulners.
+    '';
+    license = with licenses; [ asl20 ];
+    maintainers = with maintainers; [ jk developer-guy ];
```
Comment (ajs124) [inline pkgs/tools/admin/gosbom/default.nix:78]: do these people want to be added as maintainers here?

also, the commit message does not match contributing.md

**Comment by Janik-Haag** [comment]
@gitworkflows why did you mark the conversation above as resolved without answering?

**Comment by Janik-Haag** [comment]
Closing this since the author of the pr doesn't respond :/
@gitworkflows feel free to reopen this once you have time to work on it :D

**Pull request closed without merging**